index.bs

```diff
@@ -10366,13 +10368,11 @@ is the [=interface=]:
         |I|.
 </ol>
 
-If the internal \[[Call]] method
-of the [=interface object=]
-returns normally, then it must
-return an object that implements interface |I|.
+If evaluating the [=function object=] |F| returns normally,
+then it must return an object that implements interface |I|.
 This object also must be
 associated with the ECMAScript global environment associated
-with the interface object.
+with |I|.
```

Comment:
Bundled that up as an `Assert:` statement. You might have had something completely different in mind--LMK.

Also unsure if said assert should operate on `|R|` or on the result of converting `|R|` to ES (last step of the algo).